test/databaseTest.js

```diff
@@ -0,0 +1,31 @@
+var Database = require('../lib/Database');
+var query;
+
+describe('Database', function () {
+  Database = Database.extend({
+    init: function() {}
+  });
+  query = new Database();
+  it('should prepare query segment for order when with array content', function () {
+    var order = [
+      ['created'],
+      ['ormy', 'desc'],
+      ['label', 'asc']
```

Comment:
added for both string and array validation

string : https://github.com/lighterio/ormy/pull/10/files#diff-024b0d91f5be2c2a6a92ba02162e4ee3R73

array : https://github.com/lighterio/ormy/pull/10/files#diff-024b0d91f5be2c2a6a92ba02162e4ee3R50